server/www/templates/posts/detail.html

```diff
@@ -1,91 +1,117 @@
-            </button>
-        </p>
+                    <div ng-if="post.allowed_privileges.indexOf('update') !== -1" class="form-field select">
+                        <div class="custom-select init fa-user">
+                            <select 
+                                ng-change="publishPostTo()" 
+                                ng-model="publishRole"
+                            >
+                               <option 
+                                  value="draft"
+                                  translate>
+                                      post.publish_for_you
+                              </option>
+                              <option
+                                  ng-repeat="role in availableRoles"
+                                  data-icon="fa-user"
+                                  value="{{role.name}}"
+                                  >
+                                      {{role.display_name}}
+                               </option>
+                               <option 
+                                  data-icon="fa-globe"
+                                  value=""
+                                  translate>
+                                      post.publish_for_everyone
+                                </option>
 
-    </aside>
-
-    <div class="main-col">
-        <article role="article">
-            <span class="meta-data has-categories"><span class="date">{{post.updated || post.created | date:'longDate'}}</span> {{post.updated || post.created | date:'shortTime'}} <!-- via SMS --></span>
-
-            <ul class="categories-list-inline" ng-show="!!post.tags.length">
-                <li class="category" ng-repeat="tag in post.tags">
-                    <i class="fa fa-{{tag.icon}}" style="color:{{tag.color}}"></i> {{tag.tag}}
-                </li>
-            </ul>
-
-            <h1 class="post-header">{{post.title}}</h1>
-
-        	<div ng-if="post.content" class="post-content" sd-model-to-html="post.content">
-        	</div> <!-- end .post-content -->
-
-            <ul class="post-values list-unstyled">
-                <li
-                    ng-repeat="(key, value) in post.values" ng-if="form_attributes[key] && showType(form_attributes[key].type)">
-                    <post-value
-                        key="key"
-                        value="value"
-                        attribute="form_attributes[key]"
-                    ></post-value>
-                </li>
-            </ul><!-- end .post-values -->
-
-            <h3 ng-if="mapDataLoaded" translate>post.location</h3>
-            <div ng-if="mapDataLoaded">
-                <leaflet id="post-map" class="map" defaults="defaults" geojson="geojson" center="center" layers="layers"></leaflet>
+                            </select>
+                        </div>
+                        <label><span class="nodisplay">Who</span> <span translate>post.can_see_this_post</span></label>
```

Comment:
why is this split over 2 strings? This need to be all 1 string for translations to make sense.
If we need a hidden string for screenreaders we should probably store it as a complete separate string.. ie. "can see this post" and "who can see this post" as 2 completely separate strings..